src/touch/touch.rs

```diff
@@ -42,6 +44,7 @@ macro_rules! local_tm_to_filetime(
     })
 );
 
+#[cfg(unix)]
```

Comment:
Because the other function is now gone, this macro actually isn't used anywhere now, so it can be deleted too.

Reply:
👍